fabric-api-base/src/main/java/net/fabricmc/fabric/api/util/TriState.java

```diff
@@ -17,14 +17,76 @@
 package net.fabricmc.fabric.api.util;
 
 public enum TriState {
-	FALSE,
-	DEFAULT,
-	TRUE;
+	TRUE(true) {
```

Comment:
You know we wrote some tristate extensions that are now in upstream...

Reply:
sure k will diff